api/app/src/command/medical/patient/consolidate-data.ts

```diff
@@ -154,4 +175,11 @@ export const resourceTypeForConsolidation = [
   "Specimen",
 ] as const;
 
+export const resourcesSearchableBySubject = ["AdverseEvent", "RiskAssessment", "Task"] as const;
```

Comment:
`AdverseEvent` and `Task` need to be added to the [docs](https://docs.metriport.com/medical-api/api-reference/patient/get-patient-consolidated)

Reply:
Good catch!